commands.go

```diff
@@ -49,7 +49,11 @@ func forCommand(slackCommandArgs []string, slackRequest slackRequest) (*slackRes
 
 	var r slackResponse = NewEphemeral()
 
-	user := slackCommandArgs[0]
+	var user string = slackCommandArgs[0]
+	if strings.ToLower(user) == "me" {
+		user = slackRequest.Username
```

Comment:
it might need an `@me` since later I remove the `@`, like `fmt.Sprinf("@%s", user)`... I'm not sure the username in the slack request contains the `@` already. I'll check!

Reply:
Docs says the username has no `@`, so this should be:
```
if strings.ToLower(user) == "me" {
	user = fmt.Sprintf("@%s", slackRequest.Username)
}
```